salt/states/boto_elb.py

```diff
@@ -278,6 +283,19 @@ def present(
         name of pillar dict that contains alarm settings.   Alarms defined for this specific
         state will override those from pillar.
 
+    listener_policies:
+        list of dict cookie policies for http/https listeners
+        - listener_policies:
+            - listener: 80
+              policy: LBCookieStickinessPolicy|AppCookieStickinessPolicy
+              cookie: app_cookie_name|lb_cookie_expiration_period
+
+    tags
+        dict of tags
+        Example:
```

Comment:
You need to add `.. code-block:: yaml` formatting to this example so it will render properly via `rst` on our docs site.